Makefile

```diff
@@ -68,6 +68,10 @@ build: ## Build the project
 	go build -ldflags="$(LDFLAGS)" -o bin $(WHAT)
 .PHONY: build
 
+.PHONY: build-all
+build-all: WHAT := ./cmd/...
```

Comment:
just curious, is it necessary to set WHAT here?

Reply:
yes, it overrides the value coming from the command line, to always enforce all commands being built for e2e tests (which makes sense as we call those binaries).

Reply:
ah, I see